dspace/config/spring/api/task-scheduler.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:task="http://www.springframework.org/schema/task"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+    http://www.springframework.org/schema/beans/spring-beans.xsd
+    http://www.springframework.org/schema/task
+    http://www.springframework.org/task/spring-task.xsd">
+
+  <!-- Configure the scheduler -->
+  <task:scheduler id="myScheduler" pool-size="10"/>
+
+  <!-- Configure parameters -->
+  <task:scheduled-tasks scheduler="myScheduler">
+    <task:scheduled ref="GenerateSitemaps" method="generateSitemapsScheduled" cron="0 15 1 * * ?"/>
```

Comment:
This needs to be configurable, as we shouldn't assume that everyone needs this to run at the same rate.  We also should document the syntax expected here -- it looks like the syntax is defined here? https://www.quartz-scheduler.org/api/2.3.0/org/quartz/CronTrigger.html

Maybe we should create a config in `dspace.cfg` named `sitemap.cron` and changing this entry to say `cron="${sitemap.cron}"`?  
```
# Define cron for how frequently the sitemap should refresh.
# Defaults to running daily at 1:15am
# Syntax is defined at https://www.quartz-scheduler.org/api/2.3.0/org/quartz/CronTrigger.html
sitemap.cron = 0 15 1 * * ?
```

Or, if we feel a cron syntax is too complex, we could switch this to using `fixed-rate` and have a simple configuration like:
```
# Refresh sitemap every 4 hours (4 x 60 x 60 x 1,000ms)
sitemap.refreshrate = 28800000
```

I don't feel strongly here, but I suspect users might have an easier time configuring the `refreshrate` instead of a custom cron schedule.

Reply:
it would be also useful to include the ping to the search engine as a configurable option similar to how now can be specified with the `-p` from the command line.

Reply:
I did assume that spring config sufficed for it to be configurable
Moving it to dspace.cfg makes it easier to override it in local.cfg, but there are many things configured in spring which are more likely to be adjusted but only possible in the Spring config
Is there a guideline about which spring config is handled in dspace.cfg?

Reply:
@benbosman : generally, anything that is more likely to need customization (and a cronjob type of schedule would) I tend to recommend moving to DSpace configuration, so that it can be updated in `local.cfg`.  Spring Configuration is OK for *advanced* configurations which we don't want most sites to modify.

In this scenario, I do think the sitemap generation schedule will often need customization (based on size of site and other cronjobs that folks might have running).  

That said, I don't feel that we need to make the `-p` option configurable.  Instead, I'd recommend we just make the default here to use the `-a` option, which will automatically ping any search engines configured in `sitemap.engineurls`...so that is already easily configurable.

Reply:
This has been adjusted